src/lib/messenger/get-stylesheet.ts

```diff
@@ -14,7 +14,8 @@ const getStyles = (
 ): string[] => {
 	const result: string[] = [];
 	for (let i = 0; i < styleSheets.length; i += 1) {
-		const ownerNode = styleSheets[i].ownerNode;
+		// eslint-disable-next-line @typescript-eslint/no-unnecessary-condition -- Fixes noUncheckedIndexedAccess error
```

Comment:
I don't think we'll need these comments after Chris' PR, as he's temporarily set the `no-unnecessary-condition` errors to be warnings until we've finished this work: 

https://github.com/guardian/commercial/pull/1071/files

Reply:
Yep good shout! I've rebased now so that I can remove the comments